images/varnish-drupal/drupal.vcl

```diff
@@ -19,14 +20,24 @@ acl purge {
       "192.168.0.0"/16;
 }
 
+sub vcl_init {
+    new www_dir = dynamic.director(
+      port = "${VARNISH_BACKEND_PORT:-8080}",
+      first_byte_timeout = 90s,
+      between_bytes_timeout = 90s,
+      whitelist = purge,
```

Comment:
I thought that the internal pod IPs would always be in the purge list, but now that you mention it, it would be better to remove it.  5d52ecd4d176d47de9449fe0964781e33a7f4e58